docs/topics/async.txt

```diff
@@ -91,10 +90,23 @@ Detailed notes can be found in :ref:`async-queries`, but in short:
 * ``async for`` is supported on all QuerySets (including the output of
   ``values()`` and ``values_list()``.)
 
+Django also supports some asynchronous model methods that use the database::
+
+    async def updateBook(...):
+        book = await author.books.afirst()
+        book.pages = 20
+        await book.asave()
```

Comment:
Always use snake_case for Python function names as per pep 8: https://peps.python.org/pep-0008/#function-and-variable-names

Also your example was not transaction-safe, so we should not recommend it. Let’s stick with a simple create (passing `using` to make it a case that `Model.objects.acreate` cannot handle)

```suggestion
    async def make_book(...):
    	book = Book(...)
    	await book.asave(using="secondary")
```